ddl/modify_column_test.go

```diff
@@ -895,5 +890,5 @@ func TestModifyColumnTypeWhenInterception(t *testing.T) {
 	require.True(t, checkMiddleAddedCount)
 
 	res := tk.MustQuery("show warnings")
-	require.Len(t, res.Rows(), count)
+	require.Len(t, res.Rows(), 1)
```

Comment:
Maybe we could check the warning msg which contains warnings count.

Reply:
checkMiddleAddedCount can't pass the check in the master branch, and it's useless now, so I deleted it